consensus/misc/statefee.go

```diff
@@ -15,7 +15,7 @@ func CalcStateLimit(parent *types.WorkObject, stateCeil uint64) uint64 {
 
 	// If parent gas is zero and we have passed the 5 day threshold, we can set the first block gas limit to min gas limit
 	if parent.StateLimit() == 0 {
-		return params.MinGasLimit(parent.NumberU64(common.ZONE_CTX))
+		return params.MinGasLimit(parent.NumberU64(common.PRIME_CTX))
```

Comment:
should remove the number input to this function

Reply:
Changed it back to zone as agreed